_data/tagging-status.yml

```diff
@@ -294,22 +294,23 @@
  - name: algorithm
    ctan-pkg: algorithms
    type: package
-   status: unknown
+   status: currently-incompatible
    included-in: [arxiv5]
```

Comment:
with errata for TLC3 this should then say [TLC3,arxiv5].  The prio can stay at 4 as it is only a mentioning in TLC

Reply:
Done. Should it be added to algorithmic too?

Reply:
my understanding is that algorithmic is really a sub package that normally not used on its own (though it can). Anyway, in TLC it is only a remark on page I-322 and I have now changed there from algorithms to algorithm, so not  it should not get TLC3 in included-in

Reply:
Neither one loads the other and they can be used separately. algorithm just defines the float `algorithm`, algorithmic defines the `algorithmic` environment that typesets the algorithn. But point taken about tlc3